testsuite/openmodelica/interactive-API/getClassNames.mos

```diff
@@ -3,5 +3,5 @@
 getClassNames(OpenModelica);
 
 // Result:
-// {$Code,threadData,Internal,Scripting,UsersGuide,AutoCompletion}
+// {$Code,threadData,oms_getVersion,oms_setLogFile,Internal,Scripting,UsersGuide,AutoCompletion}
```

Comment:
Are we planning to add all OMSimulator API at top level? I thought it will be part of scripting. If not at least make a package `OMSimulator`.

Reply:
@adeas31 I will move it under Scripting